src/Components/Artwork/Badge.tsx

```diff
@@ -14,9 +14,14 @@ const MIN_IMAGE_SIZE = 150
 
 class Badge extends React.Component<BadgeProps> {
   get stackedLayout() {
+    // During the SSR render pass we don't have access to window pixel data so
+    // default to high density screen.
+    const devicePixelRatio =
+      typeof window === "undefined" ? 2 : window.devicePixelRatio
```

Comment:
Not that it really matters, but could we do this:

```ts
const devicePixelRatio = window?.devicePixcelRatio ?? 2
```

I guess that would require the babel plugins for safe navigation and coalescing or whatever it's called?

Reply:
Ah, tried this one :) `window` is as special case, where the `?` operator doesn't work as expected

Reply:
haha darn